react/core/main.tsx

```diff
@@ -318,6 +322,7 @@ export {
   RenderContextConsumer,
   TreePathContextConsumer,
   LazyImages as ExperimentalLazyImages,
+  useLazyImagesContext as useExperimentalLazyImagesContext,
```

Comment:
```suggestion
  useLazyImagesContext,
```

Reply:
@klzns it didn't originally have the `Context` suffix, but I added it because it might be misleading--all it does is to give a context telling whether it should lazy load images or not

Reply:
simply `useLazyImages` might imply that it does some heavy lifting in lazy loading images

Reply:
regarding example, there will be an example in the `InfoCard` component. i don't intend this API to be made public yet though. that's why I added the `Experimental` prefix, btw

Reply:
We need to go back to all of those experimental features and decide what we will do with them. It feels like we are creating many experimental things and not coming back to them, not either removing or promoting them.

Reply:
Maybe keeping a list of them

Reply:
💯